frontend/src/tests/application-tests.js

```diff
@@ -37,6 +89,13 @@ export function applicationsTests({ apiGET, apiPOST }) {
             let resp = await apiGET(`/debug/active_user`);
             expect(resp).toHaveStatus("success");
             adminUser = resp.payload;
+            // Create taonlyuser
+            resp = await apiPOST("/debug/users", taOnlyUser);
+            expect(resp).toHaveStatus("success");
+
+            surveyWithoutTranscript.answers.position_preferences = {
+                [position.position_code]: 3,
+            };
```

Comment:
I am not sure I understand the purpose of this change in the `beforeAll` ? Is that the check on line 159?